src/OpenTelemetry/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+* Fix handling of `null` values for tags in counters. Previously, when the
```

Comment:
Not just counter. this is applicable to every type of instrument.

Reply:
Agreed, pushed an update